homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,17 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> result {};
+    
+    if(count <= 0)
+    {
+        return result;
+    }
```

Comment:
You return twice result. Try to think how you could avoid this and simplify the solution.

Reply:
I will add you hint, that for loop already have condition, which validate it.